torchci/clickhouse_queries/vllm/ci_reliability/query.sql

```diff
@@ -29,6 +30,30 @@ WITH build_jobs AS (
         AND tupleElement(build, 'started_at') IS NOT NULL
         AND tupleElement(build, 'started_at') >= {startTime: DateTime64(3) }
         AND tupleElement(build, 'started_at') < {stopTime: DateTime64(3) }
+        -- Job group filtering: AMD, Torch Nightly, or Main
+        AND (
```

Comment:
Let me think a bit more about this part, this seems fragile because (1) there are other groups like Intel, TPU, etc; and (2) you need to copy the same check across multiple queries.  The use of `jobGroups` parameter makes sense, let's see if there is away to just change this parameter and don't need to change the SQL queries